.github/workflows/review.yml

```diff
@@ -24,13 +24,13 @@ jobs:
         env:
           DOCKER_IMAGE_NAME: ${{ env.REPO_NAME }}-review
           DOCKER_BUILD_ARG_REACT_APP_ENVIRONMENT: 'review'
-          DOCKER_BUILD_ARG_REACT_APP_PROFILE_LINK: "https://profiili.test.kuva.hel.ninja/loginsso"
-          DOCKER_BUILD_ARG_REACT_APP_OIDC_AUTHORITY: "https://api.hel.fi/sso-test/"
-          DOCKER_BUILD_ARG_REACT_APP_OIDC_CLIENT_ID: "https://api.hel.fi/auth/jassari"
+          DOCKER_BUILD_ARG_REACT_APP_PROFILE_LINK: "https://profiili.${{ env.BASE_DOMAIN }}/loginsso"
+          DOCKER_BUILD_ARG_REACT_APP_OIDC_AUTHORITY: "https://tunnistamo.${{ env.BASE_DOMAIN }}"
```

Comment:
Don't know if it matters or not, but this seems to be missing a `/` at the end.

Reply:
Probably matters, fixed.